frontend/src/app/app.module.ts

```diff
@@ -8,17 +8,33 @@ import { AngularFireModule } from '@angular/fire';
 import { AngularFirestoreModule } from '@angular/fire/firestore';
 import { environment } from '../environments/environment';
 import { ImagesComponent } from './images/images.component';
+import { ClarityModule } from '@clr/angular';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { RouterModule, Routes } from '@angular/router';
+import { HomeComponent } from './home/home.component';
+import { ImagesModule } from './images/images.module';
+
+const appRoutes: Routes = [
```

Comment:
we should move these route definitions to the `app-routing.module` or remove that routing module. 

Although, for best practices and to keep a good separation of concerns, let's use a separate routing module for this. 

Dude! Props to adding routing so early in the game. Thats awesome!!!